app/services/builders/chartkick/base.rb

```diff
@@ -11,14 +11,14 @@ def initialize(entity_id, query)
       def build_data(metrics)
         metrics.inject({}) do |hash, metric|
           hash.merge!(
-            metric.value_timestamp.strftime('%Y-%m-%d').to_s => value_in_hours_for(metric)
+            metric.value_timestamp.strftime('%Y-%m-%d').to_s => metric.value_as_hours
```

Comment:
maybe this could help? https://api.rubyonrails.org/classes/ActionView/Helpers/DateHelper.html#method-i-distance_of_time_in_words